### ISSUE TITLE
bin/setup fails with "/usr/local/share/chruby/chruby.sh: No such file or directory"

### ISSUE THREAD

**Issue by technicalpickles** [thoughtbot/shoulda-matchers#1482]
I'm following the CONTRIBUTING.md, and tried running setup:

```
$ bin/setup
bin/setup: line 231: /usr/local/share/chruby/chruby.sh: No such file or directory
```

The code in question is https://github.com/thoughtbot/shoulda-matchers/blob/4eb5bb0b350bb75b3c298f8e4acd1fc362f889ab/bin/setup#L231

I'm not sure if there's an expectation of where chruby is installed, but I'm on macOS on an M1 which keeps homebrew-installed stuff in /opt/homebrew instead of /usr/local

**Comment by mcmire** [comment]
Fixed by #1483, I'm assuming?

**Comment by technicalpickles** [comment]
That's right, thanks for merging!
